apis/kueue/webhooks/clusterqueue_webhook.go

```diff
@@ -158,7 +158,11 @@ func validateFlavorQuotas(flavorQuotas kueue.FlavorQuotas, coveredResources []co
 		allErrs = append(allErrs, field.Invalid(path.Child("resources"), field.OmitValueType{}, "must have the same number of resources as the coveredResources"))
 	}
 
+	coveredResNum := len(coveredResources)
```

Comment:
```suggestion
```

Reply:
Thats what I get for being lazy..  I pushed up a fix for this.